CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivities.uc

```diff
@@ -10,63 +10,156 @@
 
 class X2StrategyElement_DefaultActivities extends X2StrategyElement;
 
+
+var const string Guerilla;
+var const string Council;
+var const string SupplyRaid;
+var const string Radio;
+var const string Advent;
+var const string SupplyLift;
+var const string Resistance;
+var const string Rescue;
+var const string Ambush;
+var const string DarkEvent;
+var const string Facility;
+var const string Chosen;
```

Comment:
There is no need for these constants, just hardcode it in the individual functions.

Example from `X2StrategyElement_DefaultMissionSources`:
```uc
static function X2DataTemplate CreateSupplyRaidTemplate()
{
	local X2MissionSourceTemplate Template;

	`CREATE_X2TEMPLATE(class'X2MissionSourceTemplate', Template, 'MissionSource_SupplyRaid');
	// Some properties
	Template.OverworldMeshPath = "UI_3D.Overwold_Final.SupplyRaid_AdvConvoy";
	Template.MissionImage = "img:///UILibrary_StrategyImages.X2StrategyMap.Alert_Supply_Raid";
	// Some properties

	return Template;
}
```